src/main/java/io/kestra/plugin/git/Clone.java

```diff
@@ -34,16 +34,24 @@
 @Plugin(
     examples = {
         @Example(
-            title = "Clone a repository from http server",
+            title = "Clone a public GitHub repository",
             code = {
-                "url: https://github.com/kestra-io/plugin-template",
-                "branch: develop",
-                "username: <username>",
-                "password: <password>"
+                "url: https://github.com/dbt-labs/jaffle_shop",
+                "branch: main",
+                "username: dbt-labs"
```

Comment:
Useless username, bring confusion

Reply:
good point, I thought this is required but just checked it works without it